tests/e2e/e2e_evidence_test.go

```diff
@@ -0,0 +1,29 @@
+package e2e
```

Comment:
A more complicated scenario would be to have blocks with the following:
- DuplicateVoteEvidence
- LightClientAttackEvidence
... and check for those as well.